config/routes.rb

```diff
@@ -5,7 +5,7 @@
   # See how all your routes lay out with "rake routes".
 
   # You can have the root of your site routed with "root"
-  root 'recipes#index'
+  root 'recipes#all_recipe'
```

Comment:
does this mean that i need to create the dashboard.html.haml, dashboard_controller.rb